external/gtest.cmake

```diff
@@ -47,7 +47,7 @@ download_project(PROJ googletest
 	QUIET
 	)
 	
-	
+set(CMAKE_CXX_FLAGS ${INTERNAL_CXX_FLAGS_STR})
```

Comment:
can this be set once in the root ($REPO/Cmake) and then passed to all sub-files?